pagerduty/resource_pagerduty_event_orchestration_path_unrouted_test.go

```diff
@@ -80,32 +81,45 @@ func TestAccPagerDutyEventOrchestrationPathUnrouted_Basic(t *testing.T) {
 						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.event_action", "trigger"),
 					resource.TestCheckResourceAttr(
 						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.#", "2"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.0.name", "server_name_cpu"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.0.path", "event.summary"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.0.type", "regex"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.0.value", "High CPU on (.*) server"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.1.name", "server_name_memory"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.1.path", "event.custom_details"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.1.type", "regex"),
-					resource.TestCheckResourceAttr(
-						"pagerduty_event_orchestration_unrouted.unrouted", "sets.0.rules.0.actions.0.variables.1.value", "High memory usage on (.*) server"),
+					resource.TestCheckTypeSetElemNestedAttrs(
```

Comment:
Refactored this section and switched to `TestCheckTypeSetElemNestedAttrs` instead of `TestCheckResourceAttr` which improves the code readability a bit for the key value objects.